packages/atomic/src/components/search/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -224,7 +258,13 @@ export class AtomicCategoryFacet implements InitializableComponent {
   protected facetSearchAriaMessage!: string;
 
   public initialize() {
+    if (this.tabsIncluded.length > 0 && this.tabsExcluded.length > 0) {
+      console.warn(
+        'Values for both "tabs-included" and "tabs-excluded" have been provided. This is could lead to unexpected behavior.'
```

Comment:
Also, maybe we should update the documentation a bit

Instead of saying "this property complements ...", we should say something like "To avoid unexpected behaviors, do not set both this property and ..."